Appraisals

```diff
@@ -1,90 +1,102 @@
-if RUBY_VERSION < "2.4.0"
-  appraise "rails3-postgres" do
-    gem "test-unit"
-    gem "rails", "3.2.22.5"
-    gem "pg", platform: :ruby
-    gem 'activerecord-jdbcpostgresql-adapter', platform: :jruby
+if RUBY_VERSION < '2.4.0'
+  if RUBY_VERSION >= '1.9.1'
```

Comment:
Yes, AFAIR this is "just a hole" in the test matrix, it probably requires some versioning in Appraisal (like we do with `Rails(3|4|5)`) but will dig into it.